docs/server/commands.rst

```diff
@@ -419,6 +419,49 @@ Available options:
   (quotes included).
 
 
+.. django-admin:: set_filetype
+
+set_filetype
+^^^^^^^^^^^^
+
+.. versionadded:: 2.8
+
+This command sets file formats for projects, and also allows to convert files
+to another format.
+
+.. django-admin-option:: --from-filetype
+
+Specifies to convert stores of this file type.
+
+.. django-admin-option:: --matching
+
+Specifies which stores to convert by matching their path within the project,
+excluding file extension.
+
+`--from-filetype` and `--matching` can be both specified at the same time.
```

Comment:
I'm not sure this is needed in the docs, I'd only make a commnt if they were mutually exclusive, I'd take this for granted.